q/arrowkdb.q

```diff
@@ -132,6 +132,13 @@ pq.readParquetRowGroupsToTable:{[filename;row_groups;columns;options]
         ]
     };
 
+// ORC files
+orc.writeOrc:`arrowkdb 2:(`writeORC;4);
+orc.writeOrcFromTable:{[filename;table;options] orc.writeOrc[filename;sc.inferSchema[table];value flip table;options]};
+orc.readOrcSchema:`arrowkdb 2:(`readORCSchema;1);
+orc.readOrcData:`arrowkdb 2:(`readORCData;2);
+orc.readOrcToTable:{[filename;options] flip (fd.fieldName each sc.schemaFields[orc.readOrcSchema[filename]])!(orc.readOrcData[filename;options])};

```

Comment:
update for `WITH_NULL_BITMAP

Reply:
goes in the commit for supporting nulls in ORC files
https://github.com/KxSystems/arrowkdb/pull/27/commits/ec45934fe49f3672278054d1643c580fb4c8cefe